ducktape/cluster/remoteaccount.py

```diff
@@ -272,6 +272,16 @@ def _can_ping_url(self, url, headers):
         except Exception:
             return False
 
+    def available(self):
+        try:
+            self.ssh_client
```

Comment:
actually every time you call `self.ssh_client` in `remoteaccount` and the client already exists, it will call `send_ignore()` which will send a junk packet across the net.

Reply:
sounds good